From: Davidlohr Bueso <dave@gnu.org>
Date: Wed, 22 Feb 2012 03:06:54 +0000
Subject: [PATCH 066/105] tg3: remove IRQF_SAMPLE_RANDOM flag
Patch-mainline: v3.4-rc1
Git-commit: f274fd9a0384c448335dcc51b04c4c41caa7f432
References: bnc#790588 FATE#313912

This driver is the last user of the IRQF_SAMPLE_RANDOM flag for net drivers, and since add_*_randomness
interfaces have now deprecated the flag as a source of external noise, we can remove it.

Signed-off-by: Davidlohr Bueso <dave@gnu.org>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Nithin Nayak Sujir <nsujir@broadcom.com>
Acked-by: Benjamin Poirier <bpoirier@suse.de>
---
 drivers/net/ethernet/broadcom/tg3.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/ethernet/broadcom/tg3.c
+++ b/drivers/net/ethernet/broadcom/tg3.c
@@ -9784,7 +9784,7 @@ static int tg3_test_interrupt(struct tg3
 	}
 
 	err = request_irq(tnapi->irq_vec, tg3_test_isr,
-			  IRQF_SHARED | IRQF_SAMPLE_RANDOM, dev->name, tnapi);
+			  IRQF_SHARED, dev->name, tnapi);
 	if (err)
 		return err;
 
